src/test/java/com/wikia/webdriver/testcases/mercurytests/CuratedMainPageTests.java

```diff
@@ -173,10 +207,57 @@ public void MercuryCuratedMainPageTests_003_CheckElementsForNoTrendingArticles()
              MercuryMessages.INVISIBLE_MSG, cc.isMobileInContentVisible());
     PageObjectLogging
-        .log(PageElements.TRENDING_VIDEOS.name, MercuryMessages.VISIBLE_MSG,
-             MercuryMessages.INVISIBLE_MSG, cc.isTrendingVideosVisible());
+        .log(PageElements.TRENDING_ARTICLES.name, MercuryMessages.VISIBLE_MSG,
+             MercuryMessages.INVISIBLE_MSG, cc.isTrendingArticlesVisible());
     PageObjectLogging
         .log(PageElements.PREFOOTER.name, MercuryMessages.VISIBLE_MSG,
              MercuryMessages.INVISIBLE_MSG, cc.isMobilePrefooterVisible());
+
+    PageObjectLogging.logWarning("Check", "Element invisibility");
+    PageObjectLogging
+        .log(PageElements.TRENDING_VIDEOS.name, MercuryMessages.INVISIBLE_MSG,
+             MercuryMessages.VISIBLE_MSG, !cc.isTrendingVideosVisible());
+  }
+
+  // CCT05
+  @Test(groups = {"MercuryCuratedMainPageTests_005", "MercuryCuratedMainPageTests", "Mercury"})
+  public void MercuryCuratedMainPageTests_005_CheckElementsVisibilityExceptTrendingVideosAndArticles() {
+    BasePageObject base = new BasePageObject(driver);
+    wikiURL = urlBuilder.getUrlForWiki(MercuryWikis.MERCURY_NTAV_CC);
+    base.openMercuryArticleByName(wikiURL, MercuryArticles.NTAVCC_MAIN_PAGE);
+    CuratedPageObject cc = new CuratedPageObject(driver);
+
+    PageObjectLogging.logWarning("Check", "Elements visibility");
+    PageObjectLogging
+        .log(PageElements.TOP_LEADERBOARD.name, MercuryMessages.VISIBLE_MSG,
+             MercuryMessages.INVISIBLE_MSG,
+             cc.isMobileTopLeaderboardVisible());
+    PageObjectLogging
+        .log(PageElements.ARTICLE_TITLE.name, MercuryMessages.VISIBLE_MSG,
+             MercuryMessages.INVISIBLE_MSG, cc.isArticleTitleVisible());
+    PageObjectLogging
+        .log(PageElements.WIKIA_STATS.name, MercuryMessages.VISIBLE_MSG,
+             MercuryMessages.INVISIBLE_MSG,
+             cc.isWikiaStatsContainerVisible());
+    PageObjectLogging
+        .log(PageElements.FEATURED_CONTENT.name, MercuryMessages.VISIBLE_MSG,
+             MercuryMessages.INVISIBLE_MSG, cc.isFeaturedContentVisible());
+    PageObjectLogging
+        .log(PageElements.CURATED_CONTENT.name, MercuryMessages.VISIBLE_MSG,
+             MercuryMessages.INVISIBLE_MSG, cc.isCuratedContentVisible());
+    PageObjectLogging
+        .log(PageElements.IN_CONTENT.name, MercuryMessages.VISIBLE_MSG,
+             MercuryMessages.INVISIBLE_MSG, cc.isMobileInContentVisible());
+
+    PageObjectLogging.logWarning("Check", "Element invisibility");
+    PageObjectLogging
+        .log(PageElements.TRENDING_ARTICLES.name, MercuryMessages.INVISIBLE_MSG,
+             MercuryMessages.VISIBLE_MSG, !cc.isTrendingArticlesVisible());
+    PageObjectLogging
+        .log(PageElements.TRENDING_VIDEOS.name, MercuryMessages.INVISIBLE_MSG,
+             MercuryMessages.VISIBLE_MSG, !cc.isTrendingVideosVisible());
+    PageObjectLogging
```

Comment:
In my opinion it is hard to figure out what test does when all you can see is logging.
Could you think of some more readable way for achieving the same goal?
For instance move logging to PageObject.
I think of something like that:
```java
Boolean passed = true;
passed = cc.isMobileInContentVisible()
//all methods with assertions
passed = cc.isTrendingVideosVisible()

assertTrue(passed)
```

This way test is going to be marked as failed when it actually fails